lib/dal/src/diagram/geometry.rs

```diff
@@ -54,6 +61,14 @@ impl Geometry {
         result: DiagramResult,
     );
 
+    implement_add_edge_to!(
+        source_id: GeometryId,
+        destination_id: Ulid,
```

Comment:
Do diagram objects have their own id type? If so we should use that instead of ulid

Reply:
They don't